odpf/optimus/runtime_service.proto

```diff
@@ -194,6 +194,11 @@ service RuntimeService {
             get: "/v1/project/{project_name}/job/{job_name}/replay/{id}"
         };
     }
+    rpc GetReplayList(GetReplayListRequest) returns (GetReplayListResponse) {
```

Comment:
What if we call `ListReplays`

Reply:
`ReplayList` sounds better to me, but you have concern on this? And should we change the GetReplayStatus too?

Reply:
are we considering replay as a resource? In that case, I guess it should be `replays` in endpoint and ListReplays in the method.

Reply:
replays seems more descriptive. but i'm seeing other endpoints are not in a plural word, for example when getting projects or resource specs. thoughts?

Reply:
understood why `ListReplays` is better, just went through the method naming [convention](https://cloud.google.com/apis/design/naming_convention#method_names).

Reply:
seems our project is using singular in all the endpoint, do you think it is acceptable to keep as it is @ravisuhag ? if not, we need to apply the changes to the others as well to keep it in uniform. cc @kushsharma

Reply:
Planning to resolve this and thinking to just follow the current Optimus convention to use singular, and if a change in convention is needed, will handle it in a different issue. Please let me know if you have any concern on this, @ravisuhag